### ISSUE TITLE
Dependencies error

### ISSUE THREAD

**Issue by a-mann** [ghettovoice/vuelayers#508]
After updating open layers to [6.15](https://www.npmjs.com/package/ol)  some of the files were deleted. For example ol/source/State.js
This causes build errors
```
Module not found: Error: Can't resolve 'ol/source/State' in '/home/gitlab-runner/builds/ ... /node_modules/vuelayers/dist'
 error  in ./node_modules/vuelayers/dist/mixins.js
```

![ol git](https://user-images.githubusercontent.com/6555992/179764264-7b634d5f-ce22-4cdc-9266-26ebf3f99d63.png)

end
Can't resolve 'ol/geom/GeometryType'
Can't resolve 'ol/source/WMSServerType'
Can't resolve 'ol/extent/Corner'
Can't resolve 'ol/OverlayPositioning'

**Comment by infokartasplit** [comment]
same issue

**Comment by ghettovoice** [comment]
Hello all!
Sorry for delay.
ol team removed in v6.15 several enum types that I used in the vuelayers, htis was the issue.
The issue has fixed in v0.12.4, upgrade pls.
